tao_compiler/mlir/disc/transforms/fusion_utils.cc

```diff
@@ -1439,6 +1417,8 @@ std::unique_ptr<FusionStrategy> makeNewDeviceStrategy(StringRef device,
     return std::make_unique<StitchGpuFusionStrategy>(options);
   } else if (device == placement_utils::kCpu && strategy == "stitch_base") {
     return std::make_unique<StitchBaseCpuFusionStrategy>(options);
+  } else if (device == placement_utils::kGpu && strategy == "dot") {
+    return std::make_unique<DotGpuFusionStrategy>(options);
```

Comment:
Just wonder if there will be a `ConvGpuFusionStrategy` named `conv` in the future ?